x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/query_signals_route.test.ts

```diff
@@ -16,16 +16,21 @@ import {
 } from '../__mocks__/request_responses';
 import { requestContextMock, serverMock, requestMock, createMockConfig } from '../__mocks__';
 import { querySignalsRoute } from './query_signals_route';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { elasticsearchClientMock } from 'src/core/server/elasticsearch/client/mocks';
 
 describe('query for signal', () => {
   let server: ReturnType<typeof serverMock.create>;
-  let { clients, context } = requestContextMock.createTools();
+  let { context } = requestContextMock.createTools();
 
   beforeEach(() => {
     server = serverMock.create();
-    ({ clients, context } = requestContextMock.createTools());
+    ({ context } = requestContextMock.createTools());
 
-    clients.clusterClient.callAsCurrentUser.mockResolvedValue(getEmptySignalsResponse());
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (context.core.elasticsearch.client.asCurrentUser.search as any).mockResolvedValue(
```

Comment:
Same as above, if there's any way the mocks can be parameterized to clean up these `any`'s that would be great -- no need to happen in this PR.

Reply:
I actually found a way so I am going to attach it to this PR